src/dotnet/commands/dotnet-help/HelpCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Tools.Help
+{
+    public class HelpCommand
+    {
+        public static int Run(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                Cli.Program.PrintHelp();
```

Comment:
Should you simply move that method in here? Is it used anywhere else? Seems to me like it shouldn't. Does it currently depend on too much that is in the Cli.Program right now? If that is the case, this is fine by me, for now.